### PR TITLE
fix: log message >WARN not showing up in the UI

### PR DESCRIPTION
**Pull request by GSVarsha**
#### Issue:
Log message >WARN not showing up in the UI

Screenshot 2024-07-29 at 8 18 24 PM

#### Fix:
Revert `span.data["event"]` to `span.data["log"]`

Screenshot 2024-07-29 at 4 30 33 PM

### CODE REVIEW

File: src/instana/span/registered_span.py [hunk 1]
```diff
@@ -139,8 +139,8 @@ def _populate_local_span_data(self, span) -> None:
         if span.name == "render":
             self.data["render"]["name"] = span.attributes.pop("name", None)
             self.data["render"]["type"] = span.attributes.pop("type", None)
-            self.data["event"]["message"] = span.attributes.pop("message", None)
-            self.data["event"]["parameters"] = span.attributes.pop("parameters", None)
+            self.data["log"]["message"] = span.attributes.pop("message", None)
+            self.data["log"]["parameters"] = span.attributes.pop("parameters", None)
```
Comment (pvital) [inline src/instana/span/registered_span.py:143]: Why are you assuming it's not an event but a log?

**Review by pvital — COMMENTED** [review]
Is this only happening on the OTel branch, or should it also be adapted to the master branch?

**Comment by GSVarsha** [comment]
> Is this only happening on the OTel branch, or should it also be adapted to the master branch?

No, this happens only on OTel branch because this change was done by us during migration.

File: src/instana/span/registered_span.py (hunk 1 above)
Comment (GSVarsha) [reply src/instana/span/registered_span.py:143]: The backend expects the entry to be `span.data["log"]` so when we supply it as `span.data["event"]` it doesn’t recognise the same.

**Review by CagriYonca — APPROVED** [review]

**Review by pvital — APPROVED** [review]
It looks good to me.

**Pull request merged**